drivers/hmis/app/models/hmis/filter/enrollment_filter.rb

```diff
@@ -101,7 +101,7 @@ def with_household_tasks(scope)
       last_year_entered_before_anniversary = start_date.gt(Date.current).and(e_t[:entry_date].lt(Arel.sql(last_year_anniversary_date)))
       entered_before_anniversary = this_year_entered_before_anniversary.or(last_year_entered_before_anniversary)
 
-      scope.
+      household_tasks_scope = scope.
```

Comment:
could be more descriptive 

```suggestion
      enrollments_with_annual_due = scope.
```